bwctest.sh

```diff
@@ -2,25 +2,29 @@
 
 set -e
 
+DEFAULT_OPENSEARCH="https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.0.0/676/linux/x64/dist/opensearch/opensearch-2.0.0-linux-x64.tar.gz"
```

Comment:
They can still do that for using `yarn test:bwc` and passing the correct params. Right now people aren't pointed to run this script so ideally everything in this script is in a work in progress state.

This should be temp so it is setup to play with the CI/CD better. Ideally, this value on `main` will be set with the results from this https://github.com/opensearch-project/opensearch-build/issues/1492 so that it is always using the latest distribution of OpenSearch and we can pull the version of package.json to get the current value. Then when a release branch is cut we can just modify this so it only takes the release url something like `https://artifacts.opensearch.org/releases/bundle/opensearch/$VERSION/opensearch-$VERSION-linux-x64.tar.gz`.